composer.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "studiometa/twig-toolkit",
-  "version": "1.2.1",
+  "version": "1.2.2",
   "description": "A set of useful extension and components for Twig.",
   "license": "MIT",
   "require": {
```

Comment:
**question** Should we update php version to `"^7.3|^8.0|8.1"` ?

Reply:
The constraint from the `^` includes all minor version, this is why the package could be installed with PHP 8.1. 

Maybe we could switch to the following constraint with PHP 8.2 coming soon:

```
"php": "^7.3|~8.0|~8.1",
```

This would limit usage of the package to the following versions : 7.3.x, 7.4.x, 8.0.x, 8.1.x.

Reply:
👍